src/elements/common/base.scss

```diff
@@ -84,6 +78,16 @@
     border: 0 none;
     outline: none;
 
+    // Elements migrated to Blueprint should not use base styles
+    &:not(.bce) {
```

Comment:
this is increasing specificity which is breaking styles in shared-features. need to figure out how to exclude Explorer without increasing specificity

I don't think it's possible and we'll need to reset the overriding styles in a separate block